src/services/api/request.ts

```diff
@@ -7,6 +7,7 @@ export const authRequest = {
 
 export const dashboardRequest = {
   PROFILE: '/user/profile',
+  ORGANISATION: '/user/orgs?sort_by=role',
```

Comment:
Sorting this is supposed to be `role.name`
```suggestion
  ORGANISATION: '/user/orgs?sort_by=role.name',
```

Reply:
Ok.

Reply:
Effected